### PR TITLE
Add Media with text paragraph

### PR DESCRIPTION
**Pull request by gosia-mlynarczyk**

### CODE REVIEW

File: modules/localgov_campaigns_paragraphs/localgov_campaigns_paragraphs.module [hunk 1]
```diff
@@ -113,6 +118,33 @@ function localgov_campaigns_paragraphs_preprocess_paragraph(&$variables) {
...
+    // Check component's style.
+    $variables['style'] = $paragraph->get('localgov_media_with_text_style')->first()->value;
+
+    // Create a link.
+    if (!$paragraph->get('localgov_link')->isEmpty()) {
+      $link = $paragraph->get('localgov_link')->first()->getValue();
+      $link_options = [
+        'attributes' => [
+          'class' => 'media-with-text__link'
+        ],
```
Comment (stephen-cox) [inline modules/localgov_campaigns_paragraphs/localgov_campaigns_paragraphs.module:134]: Coding standards error

```
 133 | WARNING | [x] A comma should follow the last multiline array
     |         |     item. Found: 'media-with-text__link'
```

File: config/install/field.field.node.localgov_campaigns_page.localgov_campaigns_content.yml [hunk 2]
```diff
@@ -92,21 +94,24 @@ settings:
       localgov_link_and_summary:
         enabled: true
         weight: -25
-      localgov_quote:
+    localgov_media_with_text:
```
Comment (stephen-cox) [inline config/install/field.field.node.localgov_campaigns_page.localgov_campaigns_content.yml:97]: Looks to be an indentation error with this line.

**Review by stephen-cox — CHANGES_REQUESTED** [review]
The automated tests found a few problems.

File: modules/localgov_campaigns_paragraphs/tests/src/Functional/CampaignsParagraphsAdministrationTest.php [hunk 3]
```diff
@@ -94,6 +94,17 @@ public function testCampaignParagraphsTypes() {
...
+    // Check 'Media with text' fields.
+    $this->drupalGet('/admin/structure/paragraphs_type/localgov_media_with_text/fields');
+    $this->assertSession()->pageTextContains('localgov_title');
+    $this->assertSession()->pageTextContains('localgov_heading_level');
+    $this->assertSession()->pageTextContains('localgov_text');
+    $this->assertSession()->pageTextContains('localgov_link');
+    $this->assertSession()->pageTextContains('localgov_media_item');
+    $this->assertSession()->pageTextContains('localgov_media_position');
+    $this->assertSession()->pageTextContains('localgov_media_with_text_style');
+    $this->assertSession()->pageTextContains('localgov_localgov_opens_in_a_new_window');
```
Comment (stephen-cox) [inline modules/localgov_campaigns_paragraphs/tests/src/Functional/CampaignsParagraphsAdministrationTest.php:106]: Should be `localgov_opens_in_a_new_window'

**Review by stephen-cox — CHANGES_REQUESTED** [review]

File: modules/localgov_campaigns_paragraphs/localgov_campaigns_paragraphs.module (hunk 1 above)
Comment (gosia-mlynarczyk) [reply modules/localgov_campaigns_paragraphs/localgov_campaigns_paragraphs.module]: Fixed

File: modules/localgov_campaigns_paragraphs/tests/src/Functional/CampaignsParagraphsAdministrationTest.php (hunk 3 above)
Comment (gosia-mlynarczyk) [reply modules/localgov_campaigns_paragraphs/tests/src/Functional/CampaignsParagraphsAdministrationTest.php]: Fixed

File: config/install/field.field.node.localgov_campaigns_page.localgov_campaigns_content.yml (hunk 2 above)
Comment (gosia-mlynarczyk) [reply config/install/field.field.node.localgov_campaigns_page.localgov_campaigns_content.yml]: Fixed

**Pull request merged**